web/source/osk/bannerManager.ts

```diff
@@ -77,6 +106,16 @@ namespace com.keyman.osk {
             // Determines the image file to use for ImageBanners.
             this.imagePath = optionSpec['imagePath'];
             break;
+          case 'setPrediction':
+            let suggestions:Suggestion[] = Array();
+            for(var i=0; i<optionSpec['setPrediction'].length; i++) {
```

Comment:
This doesn't belong in `.setOptions`.  Please remove it.

If we want a way to manually feed it things for now, when active the `SuggestionBanner` is publicly accessible at `keyman.osk.banner.activeBanner`, so `keyman.osk.banner.activeBanner.updateSuggestions` can be called directly from console that way without any additional work.